pkg/codegen/docs/gen.go

```diff
@@ -1324,40 +1336,18 @@ func generateModulesFromSchemaPackage(tool string, pkg *schema.Package) map[stri
 	csharpLangHelper := getLanguageDocHelper("csharp").(*dotnet.DocLanguageHelper)
 	csharpLangHelper.Namespaces = csharpPkgInfo.Namespaces
 
-	pyLangHelper := getLanguageDocHelper("python").(*python.DocLanguageHelper)
 	scanResource := func(r *schema.Resource) {
 		mod := getMod(pkg, r.Token, modules, tool)
 		mod.resources = append(mod.resources, r)
 
-		for _, p := range r.Properties {
-			pyLangHelper.GenPropertyCaseMap(mod.pkg, mod.mod, tool, p, snakeCaseToCamelCase, camelCaseToSnakeCase)
-		}
-
-		for _, p := range r.InputProperties {
-			pyLangHelper.GenPropertyCaseMap(mod.pkg, mod.mod, tool, p, snakeCaseToCamelCase, camelCaseToSnakeCase)
-		}
+		generatePythonPropertyCaseMaps(mod, r)
 	}
 
 	scanK8SResource := func(r *schema.Resource) {
 		mod := getK8SMod(pkg, r.Token, modules, tool)
 		mod.resources = append(mod.resources, r)
 
-		// For k8s, all nested properties will use a snake_case,
-		// so we'll just add them all to the respective property
-		// case maps.
-		for _, p := range r.Properties {
```

Comment:
I fixed the underlying issue that originally prevented me from using the same logic for generating the Python property case maps. So this special logic for k8s wasn't needed anymore.